3498/3/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,25 @@
+require 'sinatra'
+require 'sinatra/activerecord'
+require 'sinatra'
+require 'sinatra/activerecord'
+require 'bcrypt'
+require_relative '../models/user.rb'
```

Comment:
make require more simple by using Ruby :) you have exact file names and paths just require in iterator